src/safeposix/syscalls/net_calls.rs

```diff
@@ -669,7 +669,7 @@ impl Cage {
                             return 0; //Already done!
                         }
 
-                        ConnState::CONNECTED | ConnState::INPROGRESS => {
+                        ConnState::CONNECTED | ConnState::CONNRDONLY | ConnState::CONNWRONLY | ConnState::INPROGRESS => {
```

Comment:
If we add CONNRDWRONLY, this is a spot for it.

Reply:
CONNRDWRONLY is the same as just CONNECTED.

Reply:
The only reason we need to include all of them in there is because of how Rust deals with enums and match.